### PR TITLE
Feature/emacoti/update ng2 module

### PR DESCRIPTION
**Pull request by emacoti**
Pull request for the issue #2

### CODE REVIEW

**Comment by emacoti** [comment]
#2

File: modules/pdb_ng2/assets/classes/scroll-loader.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  }
+
+  /**
+   * Checks to see if an element is in view.
+   *
+   * @param {object} el - element to check
+   * @returns {boolean} - in viewport
+   *
+   * @see based on SO answer: http://stackoverflow.com/a/23234031/1774183
+   * DNA-339 - fixes firefox issue and also loads if any part of element is in
```
Comment (mrjmd) [inline modules/pdb_ng2/assets/classes/scroll-loader.ts]: Let's remove references to WU tickets :)

File: modules/pdb_ng2/assets/classes/global-providers.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/**
+ * @author Shawn Stedman
```
Comment (mrjmd) [inline modules/pdb_ng2/assets/classes/global-providers.ts]: Let's remove author notes, maybe we can add acknowledgements to the README.txt to credit them instead?

File: modules/pdb_ng2/assets/classes/scroll-loader.ts [hunk 3]
```diff
@@ -0,0 +1,183 @@
+/**
+ * Created by jefflu on 1/20/2016.
```
Comment (mrjmd) [inline modules/pdb_ng2/assets/classes/scroll-loader.ts]: Let's remove author notes, maybe we can add acknowledgements to the README.txt to credit them instead?

File: modules/pdb_ng2/assets/classes/global-providers.ts (hunk 2 above)
Comment (emacoti) [inline modules/pdb_ng2/assets/classes/global-providers.ts]: Makes sense, I will add an item in the TODO.txt for this.

File: modules/pdb_ng2/assets/classes/scroll-loader.ts (hunk 3 above)
Comment (emacoti) [inline modules/pdb_ng2/assets/classes/scroll-loader.ts]: Makes sense, I will add an item in the TODO.txt for this.

**Comment by emacoti** [comment]
The comments were resolved.

**Comment by mrjmd** [comment]
Looks good, merging.

File: modules/pdb_ng2/assets/app/app.ts [hunk 4]
```diff
@@ -1,27 +1,27 @@
-import {platform, Component} from 'angular2/core';
+import {platform, Component, PACKAGE_ROOT_URL} from 'angular2/core';
 import {bootstrap, BROWSER_PROVIDERS, BROWSER_APP_PROVIDERS} from 'angular2/platform/browser';
 
 import 'rxjs/add/operator/map';
 
-import {ScrollLoader} from 'modules/pdb/modules/pdb_ng2/assets/app/scroll-loader.ts';
+import {ScrollLoader} from 'modules/custom/pdb/modules/pdb_ng2/assets/classes/scroll-loader.ts';
```
Comment (mrjmd) [inline modules/pdb_ng2/assets/app/app.ts]: Oops, this broke things because of the /custom/ subdirectory for both scroll loader and global providers.

**Pull request merged**